tests/edalize_common.py

```diff
@@ -25,8 +112,8 @@ def compare_files(ref_dir, work_root, files):
         if 'GOLDEN_RUN' in os.environ:
             shutil.copy(generated_file, reference_file)
 
-        with open(reference_file) as fref, open(generated_file) as fgen:
-            assert fref.read() == fgen.read(), f
+        assert filecmp.cmp(reference_file, generated_file, shallow=False), \
+            "Mismatch between {} and {}".format(reference_file, generated_file)
```

Comment:
Sorry, I didn't see that one before: The benefit of the `assert` is that pytest gives a diff if there's a mismatch. It would be great if we can keep that property.

Reply:
I'm not sure that's true: I think the ", f" tells pytest to just print the file name. But I'll revert the hunk: it doesn't really make much difference.